modules/manage/partials/authentication.adoc

```diff
@@ -687,6 +690,47 @@ pandaproxy_client:
 ----
 endif::[]
 
+[[plain]]
+==== SASL/PLAIN
+
+You can configure Kafka clients to authenticate using either SASL/SCRAM or SASL/PLAIN with a
+single account using the same username and password. Unlike SASL/SCRAM, which uses a challenge response with hashed
+credentials, SASL/PLAIN transmits plaintext passwords, but requires TLS for external encryption.
```

Comment:
We don't actually require TLS, we just change the log message from WARN to ERROR